configuration.go

```diff
@@ -270,6 +272,12 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 
+	//default Swarm
+	var defaultSwarm provider.Swarm
+	defaultSwarm.Watch = true
+	defaultSwarm.ExposedByDefault = true
+	defaultSwarm.Endpoint = "unix:///var/run/docker.sock"
```

Comment:
I saw some mentions of a /var/run/swarm.sock socket. Does anybody have info on that ?